### PR TITLE
clear textbox after send

### PR DESCRIPTION
**Pull request by makoteq**
Fixes  https://github.com/fossasia/susi.ai/issues/3212

Changes: change message var after click send

Demo Link: https://pr-3213-fossasia-susi-web-chat.surge.sh

Screenshots of the change: in issue

### CODE REVIEW

**Comment by makoteq** [comment]
@dipu989 Can you review?

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 1]
```diff
@@ -509,6 +509,10 @@ class Preview extends Component {
                                       })
                                     }
                                     onKeyDown={event => {
+                                      if (event.keyCode === 13) {
+                                        this.sendMessage();
+                                        this.setState({ message: '' });
+                                      }
```
Comment (akshatnitd) [inline src/components/cms/BotBuilder/Preview/Preview.js]: Please make the changes in onKeyDown function.

**Comment by makoteq** [comment]
@akshatnitd  Can you specify what's wrong?, Please

File: src/components/cms/BotBuilder/Preview/Preview.js (hunk 1 above)
Comment (makoteq) [reply src/components/cms/BotBuilder/Preview/Preview.js]: What changes? @akshatnitd

Comment (Dilshaad21) [inline src/components/cms/BotBuilder/Preview/Preview.js]: Make changes to `this.onKeydown(event)`

**Comment by makoteq** [comment]
I had to create a new repository please review This one - https://github.com/fossasia/susi.ai/pull/3268
@Dilshaad21 @akshatnitd

**Pull request closed without merging**